README.md

```diff
@@ -18,6 +18,20 @@ https://flashpaper.io
   3. Run `docker-compose up -d` to start FlashPaper
   4. Set up a reverse-proxy in front of FlashPaper that terminates SSL/TLS
 
+#### Building an image
+  You can build your own image using the provided Dockerfile in the `docker/` folder. There are currently two:
+  - For intel
+  - For ARM 64 bit
+  In order to build, move the appropriate Dockerfile to the root of the repo, and either run `docker build . -t flashpaper -f arm.Dockerfile` for example, or
+  include it in your docker-compose:
+  ```
+  build:
+      context: .
+      dockerfile: ./arm.Dockerfile
```

Comment:
Since x86_64 is more common, please change this line to `dockerfile: ./docker/Dockerfile`